.github/workflows/release.yml

```diff
@@ -62,34 +72,44 @@ jobs:
           GA4_SECRET: ${{ secrets.GA4_SECRET }}
           VITE_IS_BETA: ${{ github.event.release.prerelease == true }}
 
+      - name: 🔔 Create deployment notification
+        uses: agrc/service-now-worknote-action@v1
+        with:
+          repo-token: ${{ github.token }}
+          username: ${{ secrets.SN_USERNAME }}
+          password: ${{ secrets.SN_PASSWORD }}
+          instance-name: ${{ secrets.SN_INSTANCE }}
+          table-name: ${{ secrets.SN_TABLE }}
+          system-id: ${{ secrets.SN_SYS_ID }}
```

Comment:
Should this go into a separate job that is dependent on the mac and windows deploy jobs?

Reply:
It may also be good to add https://github.com/agrc/release-issue-notifications-action to the newly created job.